### ISSUE TITLE
verify dev.nubank

### ISSUE THREAD

**Issue by philomates** [clojars/administration#35]
Hello, I'm one of the owners of the `nubank` group. I tried to create a new project in the group and noticed there are new verification requirements. Since the group name doesn't fit the protocol of a reverse-domain name, I imagine we'll need to move to a group name that does, right?

In that case I'd like to create and verify the `dev.nubank` group.

I am requesting verification of a new group. The group name is: `dev.nubank`

- [x] This group is a [reverse-domain-based group](https://github.com/clojars/clojars-web/wiki/Verified-Group-Names#faq)
- [x] I have added a TXT DNS record for the domain that has a value of `clojars <your clojars username>`

```
$ dig -t txt nubank.dev
; <<>> DiG 9.10.6 <<>> -t txt nubank.dev
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 37812
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1
;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;nubank.dev.			IN	TXT
;; ANSWER SECTION:
nubank.dev.		86400	IN	TXT	"v=spf1 -all"
nubank.dev.		86400	IN	TXT	"clojars phillipmates"
;; Query time: 262 msec
;; SERVER: 10.0.5.13#53(10.0.5.13)
;; WHEN: Mon Apr 19 10:55:46 -03 2021
;; MSG SIZE  rcvd: 96
```
(in case it doesn't show, I think `dig -t txt nubank.dev +trace` shows some entries where the cache is up-to-date)

Thanks!

**Comment by tobias** [comment]
You are correct, **new** projects will need to be deployed to a verified group. Any **existing** project that is in a non-verified group can have new versions deployed now and in the future. 

The group https://clojars.org/groups/dev.nubank has been created & verified and you should now see the "verified" badge when you view it on Clojars. You can delete the TXT record if you like.
